department-of-veterans-affairs/developer-portal
# Add prop to crisis line modal to set initial focus on first link

a small tweak to the initial focus in the crisis line modal for [API-170](https://vajira.max.gov/browse/API-170). sets focus on the first link rather than the close button

### Testing

- verify the modal has focus, as indicated by the gold outline, on the first list item link (used to be on the close button). looks like this:

<img width="406" alt="ModalWithCustomInitialFocus170" src="https://user-images.githubusercontent.com/3241493/83897638-d526b880-a723-11ea-88f4-9a6b3427e786.png">

- check that keyboard focus still works as expected. you should be able to tab forwards through the list items, the link at the bottom, then back to the close button at the top, and you should be able to shift-tab back to the close button and loop back to the bottom link and then the list item links in reverse order.
